package.json

```diff
@@ -37,8 +37,8 @@
     "prism-react-renderer": "^1.1.1",
     "prismjs": "^1.21.0",
     "prop-types": "^15.7.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^17.0.1",
```

Comment:
are there any concerns with us upgrading to react 17?

Reply:
Shouldn't be! Check out the preview! https://pr-1160.dlyi50rq9kt6c.amplifyapp.com/

Its fully backwards compatible, so as far as I'm aware, nothing should break as a result of the upgrade.